app/views/html_publication/show.html.erb

```diff
@@ -46,7 +46,14 @@
 <%= render "shared/history_notice", content_item: content_item %>
 
 <% if content_item.withdrawn? %>
-  <%= render "govuk_publishing_components/components/notice", content_item.withdrawal_notice_component %>
+  <% withdrawn_time_tag = tag.time(display_date(content_item.withdrawn_at), datetime: content_item.withdrawn_at) %>
```

Comment:
Not relevant to this exact line, but could you do the localisation copying in a separate commit?